src/ajax/core.cljs

```diff
@@ -15,6 +15,8 @@
   (:require-macros [ajax.macros :as m]
                    [poppea :as p]))
 
+(def default-interceptors (atom ()))
```

Comment:
Another advantage of the way you've done it is that it would be possible to remove the damn things later. Maybe keep it.